---
title: "setup"
bg: green
blur: rgb(230, 255, 230)
color: black
fa-icon: cogs
fa-color: orange
---

DoriTool allows performing a functional '_in silico_' analysis at

1) Mutation/variant level, performing annotation of a set of mutations/variants, reporting eQTL results from GTEx and providing their linkage disequilibrium (LD) proxies.

2) Gene level, performing annotation of the genes tagged by the set of input variants and reporting also their Gene Ontology.

3) Pathway level.

4) Network level.

--------------------

![Doritool Logo](../img/workflow.png)


Schematic diagram of the DoriTool architecture. The workflow starts with an input file (VCF format, list of chromosome positions of the variants and their alleles or list of variant rs identifiers).

1) Variant Effect Predictor annotates the input variants with the tagged gene, the impact and some annotations described in the main text, querying Ensembl database. The next step involves locating the variants in their corresponding cytobands, and the user has the option to query the GTEx database locally by providing a tissue-specific file with the eQTLs.

2) DoriTool uses topGO (FGNet), g:Profiler and DAVID (FGNet) R packages to produce Gene Ontology terms, functional networks, enriched pathways and other annotations.

3) The main output files are the CompleteReport that contains all the annotations at transcript level provided by VEP, the SummaryReport that includes only felected information from the former, and collapsing the results by gene and effect in the transcript.

4) As an option, the user may ask for Linked variants in a window size of 500 Kb, with an LD cutoff specified in the main call.

5) In case the user has asked the previous task, a new CompleteReport profiling the linked variants will be produced.

--------------------

# <i class="fa fa-play"> Quick start</i>

1) Go to GitHub Repository  at <https://github.com/doritool> to download...

 DoriTool uses _GRCh37_ human assembl.

2) **input data** a mutation/variant call format file (VCF) or an rs identifier SNP list.

3) Perform the functional in silico analysis from the shell as follows bellow

3.1.  Variant annotation analyslis (without Linkage Disequilibrium and eQTLs).

There are two ways to specify the input file (`--  input`,  `-i file` )

`./code/DoriTool.sh -i web.rs`

`./code/DoriTool.sh -- web.rs`

3.2. Variant annotation analysis with linkage Disequilibrium (`--LD` , `-l`). Notice the threshold must be specified.

`./code/DoriTool.sh -l 0.90 -i web.rs`

`./code/DoriTool.sh --LD 0.90 -i web.rs`

3.3. Variant annotation analysis with eQTLs (`--GTEx` , `-e`). Notice that the specific human tissue required must be downloaded previously from
<https://www.gtexportal.org/home/>

`./code/DoriTool.sh --GTEx Brain_Caudate_basal_ganglia_Analysis.nominal.filtered.txt -i web.rs`

`./code/DoriTool.sh -e Brain_Caudate_basal_ganglia_Analysis.nominal.filtered.txt -i web.rs`

3.4.Variant annotation analysis with Linkage Disequilibrium and eQTLs

    ./code/DoriTool.sh -l 0.90 -e Brain_Nucleus_accumbens_basal_ganglia_Analysis.nominal.filtered.txt -i web.rs.
