<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',1],['SNV',19]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['insertion',1],['SNV',19]]));

            google.visualization.events.addListener(var_class_pie, 'select', function() {
              var_class_table.setSelection(var_class_pie.getSelection());
            });
            google.visualization.events.addListener(var_class_table, 'select', function() {
              var_class_pie.setSelection(var_class_table.getSelection());
            });
          var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',5],['intron_variant',13],['upstream_gene_variant',2]]), {slices: [{color: "#ffd700"}, {color: "#02599c"}, {color: "#a2b5cd"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',5],['intron_variant',13],['upstream_gene_variant',2]]));

            google.visualization.events.addListener(var_cons_pie, 'select', function() {
              var_cons_table.setSelection(var_cons_pie.getSelection());
            });
            google.visualization.events.addListener(var_cons_table, 'select', function() {
              var_cons_pie.setSelection(var_cons_table.getSelection());
            });
          var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',31],['3_prime_UTR_variant',1],['non_coding_transcript_exon_variant',10],['intron_variant',119],['NMD_transcript_variant',7],['non_coding_transcript_variant',50],['upstream_gene_variant',34],['downstream_gene_variant',7],['TF_binding_site_variant',4],['regulatory_region_variant',8]]), {slices: [{color: "#ffd700"}, {}, {}, {color: "#02599c"}, {}, {}, {color: "#a2b5cd"}, {color: "#a2b5cd"}, {}, {color: "brown"}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',31],['3_prime_UTR_variant',1],['non_coding_transcript_exon_variant',10],['intron_variant',119],['NMD_transcript_variant',7],['non_coding_transcript_variant',50],['upstream_gene_variant',34],['downstream_gene_variant',7],['TF_binding_site_variant',4],['regulatory_region_variant',8]]));

            google.visualization.events.addListener(consequences_pie, 'select', function() {
              consequences_table.setSelection(consequences_pie.getSelection());
            });
            google.visualization.events.addListener(consequences_table, 'select', function() {
              consequences_pie.setSelection(consequences_table.getSelection());
            });
          var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',31]]), {slices: [{color: "#ffd700"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['missense_variant',31]]));

            google.visualization.events.addListener(coding_pie, 'select', function() {
              coding_table.setSelection(coding_pie.getSelection());
            });
            google.visualization.events.addListener(coding_table, 'select', function() {
              coding_pie.setSelection(coding_table.getSelection());
            });
          var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['2',2],['5',3],['7',2],['10',4],['11',8],['20',1]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['2',2],['5',3],['7',2],['10',4],['11',8],['20',1]]));

            google.visualization.events.addListener(chr_bar, 'select', function() {
              chr_table.setSelection(chr_bar.getSelection());
            });
            google.visualization.events.addListener(chr_table, 'select', function() {
              chr_bar.setSelection(chr_table.getSelection());
            });
          var chr_2_area = drawArea('chr_2_area', 'Distribution of variants on chromosome 2', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_5_area = drawArea('chr_5_area', 'Distribution of variants on chromosome 5', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_7_area = drawArea('chr_7_area', 'Distribution of variants on chromosome 7', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_10_area = drawArea('chr_10_area', 'Distribution of variants on chromosome 10', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_11_area = drawArea('chr_11_area', 'Distribution of variants on chromosome 11', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_20_area = drawArea('chr_20_area', 'Distribution of variants on chromosome 20', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',0]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',0],['10-20%',4],['20-30%',17],['30-40%',3],['40-50%',3],['50-60%',3],['60-70%',0],['70-80%',0],['80-90%',0],['90-100%',1]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
    }
    
    tr:nth-child(odd) {
      background-color: rgb(238, 238, 238);
    }
    
    td {
      padding: 5px;
    }
    
    td:nth-child(odd) {
      font-weight: bold;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 500px;
      float: right;
      display: inline;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/info/docs/variation/vep/vep_script.html"><img src="http://www.ensembl.org/img/vep_logo.png"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div> <div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li> <li><a href="#run_stats">VEP run statistics</a></li> <li><a href="#gen_stats">General statistics</a></li> <li><a href="#var_class">Variant classes</a></li> <li><a href="#var_cons">Consequences (most severe)</a></li> <li><a href="#consequences">Consequences (all)</a></li> <li><a href="#coding">Coding consequences</a></li> <li><a href="#chr">Variants by chromosome</a></li> <li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td> <td>84 (84)</td></tr> <tr><td>Cache/Database</td> <td>/home/bioinfo/.vep/homo_sapiens/84_GRCh37</td></tr> <tr><td>Species</td> <td>homo_sapiens</td></tr> <tr><td>Command line options</td> <td><pre>--cache --force_overwrite --port 3337 -i /home/bioinfo/Escritorio/doritool_pruebas/input/web.rs --symbol --tab --plugin UpDownDistance,10000,5000 --regulatory --plugin NearestGene,limit=1,max_range=1000000 --plugin Condel,/home/bioinfo/.vep/Plugins/config/ -o /home/bioinfo/Escritorio/doritool_pruebas/output/VEP/condeloutput.txt</pre></td></tr> <tr><td>Start time</td> <td>2017-06-01 09:04:24</td></tr> <tr><td>End time</td> <td>2017-06-01 09:04:32</td></tr> <tr><td>Run time</td> <td>8 seconds</td></tr> <tr><td>Input file (format)</td> <td>/home/bioinfo/Escritorio/doritool_pruebas/input/web.rs (ID)</td></tr> <tr><td>Output file</td> <td>/home/bioinfo/Escritorio/doritool_pruebas/output/VEP/condeloutput.txt <a href="/home/bioinfo/Escritorio/doritool_pruebas/output/VEP/condeloutput.txt">[text]</a></td></tr></table><h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td> <td>23</td></tr> <tr><td>Variants processed</td> <td>20</td></tr> <tr><td>Variants remaining after filtering</td> <td>20</td></tr> <tr><td>Lines of output written</td> <td>214</td></tr> <tr><td>Novel / existing variants</td> <td>-</td></tr> <tr><td>Overlapped genes</td> <td>14</td></tr> <tr><td>Overlapped transcripts</td> <td>69</td></tr> <tr><td>Overlapped regulatory features</td> <td>4</td></tr></table><hr /><h3 id="var_class">Variant classes</h3><div style="width: 800px; height: 200px" id="var_class_pie">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr /><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div style="width: 800px; height: 200px" id="var_cons_table">&nbsp;</div><hr /><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div style="width: 800px; height: 200px" id="consequences_table">&nbsp;</div><hr /><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr /><h3 id="chr">Variants by chromosome</h3><div style="width: 800px; height: 200px" id="chr_bar">&nbsp;</div><div style="width: 800px; height: 200px" id="chr_table">&nbsp;</div><hr /><h3 id="chr_2">Distribution of variants on chromosome 2</h3><div id="chr_2_area" style="width: 800px; height: 200px">&nbsp;</div><hr /><h3 id="chr_5">Distribution of variants on chromosome 5</h3><div style="width: 800px; height: 200px" id="chr_5_area">&nbsp;</div><hr /><h3 id="chr_7">Distribution of variants on chromosome 7</h3><div style="width: 800px; height: 200px" id="chr_7_area">&nbsp;</div><hr /><h3 id="chr_10">Distribution of variants on chromosome 10</h3><div style="width: 800px; height: 200px" id="chr_10_area">&nbsp;</div><hr /><h3 id="chr_11">Distribution of variants on chromosome 11</h3><div style="width: 800px; height: 200px" id="chr_11_area">&nbsp;</div><hr /><h3 id="chr_20">Distribution of variants on chromosome 20</h3><div id="chr_20_area" style="width: 800px; height: 200px">&nbsp;</div><hr /><h3 id="protein">Position in protein</h3><div style="width: 800px; height: 200px" id="protein_bar">&nbsp;</div></div>
</div></body>
</html>
